src/3d/symbols/qgspointcloud3dsymbol.cpp

```diff
@@ -0,0 +1,62 @@
+/***************************************************************************
+  qgspointcloud3dsymbol.h
+  ------------------------------
+  Date                 : November 2020
+  Copyright            : (C) 2020 by Nedjima Belgacem
+  Email                : belgacem dot nedjima at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgspointcloud3dsymbol.h"
+
+// TODO: For some reason whwn I define function on a .cpp file they don't get included in the qgis_app target
```

Comment:
This would have been caused by the missing _3D_EXPORT macro -- it should be safe to move these back now